### DISCUSSION TITLE
Allow editing of paid/free orders by customer

### DISCUSSION THREAD

**Discussion by loelkes** [pretix/pretix#1943] [Ideas]
Is it possible to allow users to edit free items in their order up to a certain date? As far I can see, user can be allowed to cancel but not to change orders. 

Our situation: for the GPN event the orders for merchandise start long before the design and brand for the tshirts is set. After publication of the design, users would like to change their order (for example to another size) or cancel parts of it (because the price changed).

We use pretix as pre-order system, items can be paid cash only. For now, all items are 'free' in the shop as we don't 'sell' anything online (there are multiple reasons). All free orders are immediately confirmed and cannot be changed...

Can this be implemented?

**Comment by raphaelm** [comment]
This would be very nice, unfortunately also very hard due to complex product structure options (bundles, addons, availability timeframes, quotas, …) that all need to be respected.
